### PR TITLE
Re-adding cordova-impl.xml

### PR DESCRIPTION
**Pull request by bbesmanoff**
This PR brings `cordova-impl.xml` back into the project.  It started with what was last there historically, and then was updated with what @thenaterhood had locally.

### CODE REVIEW

File: .sencha/app/cordova-impl.xml [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                <then>
+                    <x-echo message="[Cordova] @{text}" level="@{level}"/>
+                </then>
+                <else>
+                    <x-echo message="[Cordova] @{message}" level="@{level}"/>
+                </else>
+            </if>
+        </sequential>
+    </macrodef>
+</project>
```
Comment (thenaterhood) [inline .sencha/app/cordova-impl.xml]: There's no newline at the end of the file. Since we're poking this, we may as well add one.

Comment (bbesmanoff) [inline .sencha/app/cordova-impl.xml]: Resolved in 18e6927

**Comment by cman131** [comment]
:european_post_office: I'm good with it as it matches mine :+1:

**Comment by thenaterhood** [comment]
Since this is from me and worked for me before, I approve. :european_post_office:

**Pull request merged**